README.md

```diff
@@ -72,6 +72,21 @@ By default, the application will use `mongoqp-cache/` within the system's
 temporary directory. This path, which must be writable, may be customized via
 the `twig.cache_dir` configuration option.
 
+#### Docker Setup 
+Clone the repo
+
+You just have to add the mongo uri in config.php and run commands : 
+```
+$ docker build -t <your-docker-account>/mongoqp .
```

Comment:
What is `<your-docker-account>` supposed to be replaced with? If `docker build` supports [Git repositories](https://docs.docker.com/engine/reference/commandline/build/#git-repositories) with tags, wouldn't that be preferable?